torch/_dynamo/variables/misc.py

```diff
@@ -259,6 +318,32 @@ def call_function(self, tx, args, kwargs):
         options = VariableTracker.propagate(self, args, kwargs.values())
         return AutogradFunctionVariable(self.fn_cls, source=self.source, **options)
 
+    def call_method(
+        self,
+        tx,
+        name,
+        args: "List[VariableTracker]",
+        kwargs: "Dict[str, VariableTracker]",
+    ):
+        if name == "backward":
+            with tx.strict_translation_mode():
+                return tx.inline_call(
+                    tx, UserFunctionVariable(self.fn_cls.backward), args, kwargs
+                )
+        return tx.inline_call(
+            tx, UserFunctionVariable(self.fn_cls.forward), args, kwargs
+        )
+
+
+class SaveSimulatingAutogradFunctionContext(torch.autograd.function.FunctionCtx):
```

Comment:
We discussed this a bit, but do we still need `SaveSimulatingAutogradFunctionContext`?